lib/nanoc-conref-fs/datafiles.rb

```diff
@@ -20,7 +20,7 @@ def self.apply_conditionals(config, path:, content:, rep:)
       end
 
       path = path.dup
-      path.slice!('data/')
+      path.slice!("#{ConrefFS.data_dir_name(config)}/") # Beware the slashes, they are important for tokenization
```

Comment:
Should the slash come from the data_dir_name function, it seems like it is also added in other places this is used...

Reply:
Yuuup.  So, this actually opens up a whole can of worms that I don't think are important at this time.  I spent about 45 minutes just now rolling through and there are some tests that are doing some unique things with manipulating the data_dir, and I can't predict what else in the system might be depending on this.

Since my primary goal here was to get the custom `data_dir` working so that I could move forward with another project that's dependent on it, I'd like to table this for the time being.  :)